scripts/xnat/check_new_sessions

```diff
@@ -319,30 +319,36 @@ def incomplete_scan_check(eid, manufacturer, label):
     missing_scans = []
     sess_type = 'xnat:mrSessionData/scans/scan/type'
     scan_types = ifc.select.experiment(eid).attrs.get(sess_type)
+    scan_types_set = set(scan_types)
+    ge_scan_types = set(scan_type_dictionary['GE MEDICAL SYSTEMS'])
+    siemens_scan_types = set(scan_type_dictionary['SIEMENS'])
+
     if manufacturer == 'GE MEDICAL SYSTEMS':
-        for scantype in set(scan_type_dictionary['GE MEDICAL SYSTEMS']):
-            if scantype in set(scan_types):
-                scantype = scantype
-            else:
-                missing_scans.append(scan_types)
+        if ge_scan_types.issubset(scan_types_set):
+            return
```

Comment:
I like this approach using sets, but I think this needs to be reversed or use `set.issuperset` 

scan_types_set has all of the required scans, but ge_scan_types will include additional, non-required scans. So ge_scan_types should be a superset containing all items in scan_types_set